.github/workflows/4-deploy-to-prod.yml

```diff
@@ -78,6 +89,27 @@ jobs:
           githubToken: ${{ secrets.GITHUB_TOKEN }}
           branch: main
```

Comment:
We need to do this (the same thing, `branch: main`) in `2-deploy-to-staging.yml`, or else our staging deploy will not go out to the "production" staging domain.

Reply:
Also, could you comment both settings of `branch`, with

    # Cloudflare uses `main` as the default production branch, so we push using the `main` branch so that it can be aliased by a custom domain.